src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.html

```diff
@@ -1,22 +1,42 @@
-<ng-container *ngIf="subVolumes$ | async as subVolumes">
-  <cd-table [data]="subVolumes"
-            columnMode="flex"
-            [columns]="columns"
-            selectionType="single"
-            [hasDetails]="false"
-            (fetchData)="fetchData()"
-            (updateSelection)="updateSelection($event)">
+<div class="row">
+  <div class="col-sm-2">
+    <h2 i18n>Groups</h2>
+    <ng-container *ngIf="subVolumeGroups$ | async as subVolumeGroups">
+      <ul class="nav flex-column nav-pills">
+        <li class="nav-item">
+          <a class="nav-link"
+             [class.active]="'' === subVolumeGroupName"
+             (click)="selectSubVolumeGroup('')">Default</a>
```

Comment:
you don't need to pass empty string here because you are initializing the function parameter with an empty string in the definition.
```suggestion
             (click)="selectSubVolumeGroup()">Default</a>
```